en/developer_guide/manage_assets.rst

```diff
@@ -0,0 +1,267 @@
+            env: process.env
+        };
+        config = merge.recursive(true, config, loadDirConfig('./grunt_tasks/options/'));
+        config = merge.recursive(true, config, loadDirConfig('./vendor/open-orchestra/open-orchestra-cms-bundle/OpenOrchestra/GruntTasks/Options/'));
+
+        grunt.initConfig(config);
+    };
+
+    function loadDirConfig(path) {
+        var glob = require('glob');
+        var merge = require('merge');
+        var dirConfig = {};
+
+        glob.sync('*', {cwd: path}).forEach(function(filename) {
+            var fileConfig = loadFileConfig(path, filename);
+            dirConfig = merge.recursive(true, dirConfig, fileConfig);
+        });
+
+        return dirConfig;
+    }
+
+    function loadFileConfig(path, filename) {
+        var keys =  filename.replace(/\.js$/,'').split('.');
+
+        var buildFileConfig = function(keys, filepath) {
+            if (keys.length == 0) {
+                return require(filepath);
+            } else {
+                var subArray = {};
+                var index = keys[0];
+                keys.shift();
+                subArray[index] = buildFileConfig(keys, filepath);
+                return subArray;
+            }
+        }
+
+        return buildFileConfig(keys, path + filename);
+    }
+
+When launching grunt (via the shell command ``./node_modules/.bin/grunt``), this script will be run. It
+will search for grunt tasks in directories ``./grunt_tasks`` and ``./vendor/open-orchestra/open-orchestra-cms-bundle/OpenOrchestra/GruntTasks``.
+It will then search for tasks options in ``./grunt_tasks/options/`` and ``./vendor/open-orchestra/open-orchestra-cms-bundle/OpenOrchestra/GruntTasks/Options/``.
+As you run grunt without any parameter it will then launch the Default task.
+
+.. _add an external bundle using grunt:
+
+Adding an external bundle using grunt
+-------------------------------------
+
+If you take a closer look at the ``open-orchestra-cms-bundle``, you will see it does not contain
+the media administration part. To manage the media, you have to add the ``open-orchestra-media-admin-bundle``.
+
+This bundle comes with its own javascript and css files that need to be added to the website assets
+files. Assuming you have added the bundle normally to your symfony application, you still have to
+configure grunt to use the bundle assets.
+
+To do this you need to update 3 times your grunt configuration:
+
+- load the new tasks
+- load the new task options
+- tell somewhere when these tasks must be run
+
+To load the task, you only have to use ``grunt.loadTasks`` on the correct directory. In our case
+you only have to update the ``gruntfile.js`` file with this:
+
+.. code-block:: javascript
+
+    // line 5
+    grunt.loadTasks('./vendor/open-orchestra/open-orchestra-media-admin-bundle/OpenOrchestra/GruntTasks');
+
+To load task options, you have two possibilities, depending on what you want to do: load a single
```

Comment:
I think you should split the addition of  a single file and of a directory.

It should be easier to read